db/sqlinterfaces.c

```diff
@@ -1699,9 +1677,6 @@ static void sql_update_usertran_state(struct sqlclntstate *clnt)
     if (!sql)
         return;
 
-    while (isspace(*sql))
-        ++sql;
-
```

Comment:
Did you test this change?

Reply:
1697     const char *sql = clnt->sql;
We are imposing clnt->sql does not contain leading whitespace from all paths.